dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -333,6 +336,59 @@ object TankGenerator {
       .take(requestsCount)
   }
 
+  private def mkMassTransfers(accounts: List[JPrivateKey], requestCount: Int): List[Request] = {
+
+    val assets = mkAssets(10)
+    val initialValue = 100000000000L
+    val assetOwners = accounts.map(_ -> assets(new Random().nextInt(assets.length)))
+    val allRecipients = assetOwners.map(_._1).toSet
+
+    def mkMassTransfer(transfers: List[Transfer], asset: AssetId): MassTransferTransaction =
+      MassTransferTransaction
+        .builder(transfers.asJava)
+        .assetId(asset)
+        .fee(settings.defaults.massTransferFee + (transfers.size + 1) * settings.defaults.massTransferMultiplier)
+        .version(1)
+        .timestamp(System.currentTimeMillis() + Random.nextLong(100000))
+        .getSignedWith(issuer)
+
+    assetOwners.map { case (assetOwner, asset) =>
```

Comment:
`foreach`? we don't use the result of `map`.

Reply:
https://github.com/wavesplatform/matcher/blob/DEX-1026-sc6/dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala#L373